packages/forms/src/directives/validators.ts

```diff
@@ -579,8 +583,12 @@ export class MinLengthValidator implements Validator, OnChanges {
   }
 
   private _createValidator(): void {
-    this._validator = minLengthValidator(
-        typeof this.minlength === 'number' ? this.minlength : parseInt(this.minlength, 10));
+    this._validator =
+        this.enabled() ? minLengthValidator(toInteger(this.minlength!)) : nullValidator;
+  }
+
+  enabled(): boolean {
+    return this.minlength !== null;
```

Comment:
Just to keep things fully backwards-compatible from runtime logic perspective, I'd propose using `!=` vs `!==`. 
```suggestion
    return this.minlength != null /* both `null` and `undefined` */;
```
The `!==` is always preferable, but since that code was there (in the `validate` function) for quite some time and some people used `null` and `undefined` as a way to disable this validator, it'd be safer to preserve old comparison.